env/env.off

```diff
@@ -0,0 +1,7 @@
+# NOTE: this is an addition to .env file, not a standalone file !
+# It should be loaded with: source env/setenv.sh <env>
+
+COMPOSE_PROJECT_NAME=po_off
```

Comment:
Should we put po_off as default also in .env ?

Reply:
Done.